sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/datalake_directory_client.hpp

```diff
@@ -98,14 +98,14 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
      * @brief Create a directory. By default, the destination is overwritten and
      *        if the destination already exists and has a lease the lease is broken.
      * @param options Optional parameters to create the directory the path points to.
-     * @return Azure::Core::Response<CreateDirectoryResult> containing the information of the
-     * created directory
+     * @return Azure::Core::Response<Models::CreateDirectoryResult> containing the information of
```

Comment:
The namespaces here are inconsistent.

shouldn't it be `Azure::Core::Response<Azure::Storage::Models::CreateDirectoryResult>`?
Why did you omit `Azure::Storage::`?

Reply:
The full namespace is `Azure::Storage::Files::DataLake::Models::CreateDirectoryResult`, this line is already under `Azure::Storage::Files::DataLake::` so it makes sense to omit what's before `Models` namespace.